Resources/Prototypes/Nyanotrasen/Research/experimental.yml

```diff
@@ -12,7 +12,7 @@
   recipeUnlocks:
   - ClothingHeadHelmetInsulated
   - ClothingHeadCage
-  # - ShellSoulbreaker
+  # - ShellSoulbreaker # DeltaV - Placing it under Exotic Ammunition because that's what it is.
```

Comment:
Did you intend to leave this line commented out?

Reply:
Yeah I moved it to another research during the creation of the .38 ammo type because it fit the whole "exotic ammo".
https://github.com/DeltaV-Station/Delta-v-rebase/blob/9b91b2b2edea2124f2c0b8509c20493d6f9c8418/Resources/Prototypes/DeltaV/Research/arsenal.yml#L4-L15
It's in this PR uncommented. I can move it back to this research if wanted.